### PR TITLE
Feature 340: quadratic funding and funding distribution

### LINKED ISSUE #340: Implement QF for the leaderboard calculation of R17
The calculation for winning proposals will change for round 17.  the current implementation needs to be updated and tested properly before the start of round 17.

### PR DESCRIPTION
**Pull request by KinimodEtlon**
closes #340

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -15,9 +15,10 @@
     "start:prod": "node dist/main",
     "lint": "eslint \"{src,apps,libs,test}/**/*.ts\" --fix",
     "test": "cross-env NODE_ENV=dev jest",
-    "test:watch": "jest --watch",
-    "test:cov": "jest --coverage",
-    "test:debug": "node --inspect-brk -r tsconfig-paths/register -r ts-node/register node_modules/.bin/jest --runInBand",
+    "test:watch": "cross-env NODE_ENV=dev jest --watch",
+    "test:cov": "cross-env NODE_ENV=dev jest --coverage",
+    "test:debug": "cross-env NODE_ENV=dev node --inspect-brk -r tsconfig-paths/register -r ts-node/register ./node_modules/jest/bin/jest.js --runInBand",
+    "test:debug:mac": "node --inspect-brk -r tsconfig-paths/register -r ts-node/register node_modules/.bin/jest --runInBand",
```
Comment (marhali) [inline package.json:21]: question: is cross-env not working on mac?

**Review by marhali — APPROVED** [review]
LGTM

File: package.json (hunk 1 above)
Comment (KinimodEtlon) [reply package.json:21]: cross-env should work for mac as well, the important difference it the changed path to the jest.js file

**Review by marhali — APPROVED** [review]

File: src/airtable/mapper/dao-proposal.mapper.ts [hunk 2]
```diff
@@ -35,6 +35,14 @@ export class DaoProposalMapper {
                     ? parseInt(airtableData['OCEAN Requested'])
                     : 0,
             },
+            minimumRequestedFunding: {
+                usd: airtableData['Minimum USD Requested']
+                    ? parseInt(airtableData['Minimum USD Requested'])
```
[the remark this answers is not in the document]
Comment (KinimodEtlon) [reply src/airtable/mapper/dao-proposal.mapper.ts]: the api returns number, so no need to parse anyway

**Review by Abrom8 — APPROVED** [review]

**Pull request merged**